src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { CommentInfo } from '../CommentInfo';
+
+export const PostInfo = ({ post, user, comments }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <p>
+        {' Posted by  '}
+        <UserInfo user={user} />
+      </p>
+    </div>
+    <p className="PostInfo__body">{post.body}</p>
+    <hr />
+    {comments.length === 0 ? (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    ) : (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo key={comment.id} comment={comment} />
+        ))}
+      </div>
+    )}
+  </div>
```

Comment:
It would be better to first put the right case.
```suggestion
    <p className="PostInfo__body">{post.body}</p>
    
    <hr />
    
    {comments.length > 0 ? (
        <div className="CommentList">
        {comments.map(comment => (
          <CommentInfo key={comment.id} comment={comment} />
        ))}
      </div>
    ) : (
     <b data-cy="NoCommentsMessage">No comments yet</b>
    )}
  </div>
```